src/encoded/tests/data/workbook-inserts/experiment_hi_c.json

```diff
@@ -72,6 +72,9 @@
         "lab": "test-4dn-lab",
         "aliases": ["labsample:alias3"],
         "files": ["4DNFIO67APZ1", "4DNFIO67AZZ1"],
+        "other_processed_files": [{
+            "title": "Test FileFastq as opf",
+            "files": ["/files-fastq/4DNFIO67AZB1/"]}],
```

Comment:
Codacy found an issue: [A space is required before '}'.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6115226)

Reply:
To make the json consistent the convention would be to move the `}],` on its own line as the same level of indentation as "other_processed_files"